.travis.yml

```diff
@@ -59,26 +59,40 @@ matrix:
           compiler: gcc
 
 before_script:
-    - sh .travis-create-release.sh $TRAVIS_OS_NAME
-    - tar -xvzf _srcdist.tar.gz
-    - mkdir _build;
-    - cd _build;
+    - if [ -n "$DESTDIR" ]; then
+          srcdir=../_srcdist;
+          top=..;
+          sh .travis-create-release.sh $TRAVIS_OS_NAME;
```

Comment:
I have a semi-related question: travis-create-release runs

./Configure dist

and then below we run ./Configure again. Does this make sense?